source/timer.c

```diff
@@ -28,26 +28,30 @@ static bool set_timer_negative(Handle timer, u32 kernel_callback_int, int error)
   Result res;
   u64 kernel_callback_shifted_goal, timeout;
 
-  kernel_callback_shifted_goal = ((u64)0x80000000 + error) << 32;
+  kernel_callback_shifted_goal = ((u64)0x80000000) << 32;
   timeout = ((u64)(kernel_callback_int - 0x80000000) << 32);
+  switch (carry) {
+    case 1:
+      timeout += 0x55555556;
+      break;
+    case 2:
+      timeout += 0xaaaaaaab;
+      break;
```

Comment:
these value is actually not magic.
64 bit divided values is `0x.....55555555` or `0x.....aaaaaaaa`
so i increased just last one.
but if we can catch right timing, can chage like `0x....55560000` :)